examples/wfs.js

```diff
@@ -43,32 +43,27 @@ view.camera.camera3D.position.set(1839739, 5171618, 910);
 view.camera.camera3D.lookAt(new itowns.THREE.Vector3(1840839, 5172718, 0));
 
 // eslint-disable-next-line no-new
-new itowns.FirstPersonControls(view, { focusOnClick: true, moveSpeed: 1000 });
+new itowns.PlanarControls(view, {});
 
 // Request redraw
 view.notifyChange(true);
 
-function colorFunctionLine(layer, node, featureCollection) {
-    var i;
-    var featureProperties;
-    var rgb;
-    var colors = [];
-
-    for (i = 0; i < featureCollection.features.length; i++) {
-        featureProperties = featureCollection.features[i].properties;
-
-        rgb = featureProperties.couleur.split(' ');
-        colors.push(new itowns.THREE.Color(rgb[0] / 255, rgb[1] / 255, rgb[2] / 255));
-    }
-
-    itowns.FeatureProcessing.assignColorsToFeatureCollection(
-        featureCollection, featureCollection.children[0], colors);
+function setMaterialLineWith(mesh) {
+    mesh.material.linewidth = 5;
+}
 
-    featureCollection.children[0].material.linewidth = 5;
+function colorLine(properties) {
+    var rgb = properties.couleur.split(' ');
+    return new itowns.THREE.Color(rgb[0] / 255, rgb[1] / 255, rgb[2] / 255);
 }
 
 view.addLayer({
-    update: itowns.FeatureProcessing.update(colorFunctionLine),
+    update: itowns.FeatureProcessing.update,
+    convert: itowns.Feature2Mesh.convert({
+        onMeshCreated: setMaterialLineWith,
+        color: colorLine,
+        altitude: undefined,
```

Comment:
You don't need to explicitely set altitude to ` undefined`: you can just omit altitude.